assets/setup-wizard/features/index.js

```diff
@@ -106,19 +116,14 @@ const Features = () => {
 			return;
 		}
 
-		const wooCommerceFeature = features.find( ( f ) => wcSlug === f.slug );
-		const isWooCommerceInstalled =
-			wooCommerceFeature &&
-			INSTALLED_STATUS === wooCommerceFeature.status;
-
 		if (
 			needWooCommerce &&
 			! isWooCommerceSelected &&
-			! isWooCommerceInstalled
+			! isWooCommerceInstalled()
 		) {
 			setSelectedSlugs( ( prev ) => [ ...prev, wcSlug ] );
 		}
-	}, [ getSelectedFeatures, features ] );
+	}, [ getSelectedFeatures, features, isWooCommerceInstalled ] );
```

Comment:
I think now we could remove the `features` from here, right?

Reply:
Yup removed in d61d5a3